### PR TITLE
Preparare for 2020.04 snapshot

### PR DESCRIPTION
**Pull request by pyhalov**

### CODE REVIEW

File: components/openindiana/release/files/release [hunk 1]
```diff
@@ -1,4 +1,4 @@
-             OpenIndiana Hipster 2019.10 (powered by illumos)
+             OpenIndiana Hipster %%DISTRIBUTION_VERSION%% (powered by illumos)
```
Comment (pyhalov) [inline components/openindiana/release/files/release]: Note, first line in motd file format is used at least by installer to determine URL to release notes, by openindiana-welcome to get distribution version and perhaps by some other software, so we can't just change it to '$(DISTRIBUTION_NAME) $(DISTRIBUTION_VERSION)'

File: make-rules/shared-macros.mk [hunk 2]
```diff
@@ -100,6 +100,9 @@ space := $(empty) $(empty)
 
 ROOT =			/
 
+# Distribution version
+DISTRIBUTION_NAME = OpenIndiana
```
Comment (pyhalov) [inline make-rules/shared-macros.mk]: So, question is if we should include 'Hipster' in DISTRIBUTION_NAME :)

Comment (Toasterson) [reply make-rules/shared-macros.mk]: Yes I would say so. It makes us stand out from the older Release based OpenIndiana versions. Or we could follow OmniOS here.

**Comment by Toasterson** [comment]
Will you spin images once this is merged? I will spin Vagrant boxes as soon as we have images.

**Comment by pyhalov** [comment]
Yes, if testing them doesn't find any problems, I'll announce snapshot.

**Comment by AndWac** [comment]
Am 03.05.20 um 14:42 schrieb Alexander Pyhalov:
>
> Yes, if testing them doesn't find any problems, I'll announce snapshot.
>
If you can provide a link I can test iso's, too.

Regards
Andreas

**Comment by pyhalov** [comment]
So far I have no isos, have to rebuild illumos-gate after this PR

**Comment by AndWac** [comment]
Is see. If you want me to test just write me an email.

What do you think about #5706? Despite the fact that he still needs to
squash it seems to be ok for me. I still need to check both VMs again,
tough.

Am 03.05.20 um 18:25 schrieb Alexander Pyhalov:

**Pull request merged**